backend/api/Areas/Admin/Models/Parcel/BuildingModel.cs

```diff
@@ -30,6 +31,16 @@ public class BuildingModel : Model.BaseModel
 
         public string BuildingPredominateUse { get; set; }
 
+        public int BuildingOccupantTypeId { get; set; }
+
+        public string BuildingOccupantType { get; set; }
+
+        public DateTime LeaseExpiry { get; set; }
```

Comment:
This should be nullable

Reply:
Right, there are models then just the ones I'm using.